src/frame.rs

```diff
@@ -0,0 +1,82 @@
+use std::io::ErrorKind;
+use std::io::Read;
+
+use header::{parse_frame_header, Bitrate, FrameHeader, Layer};
+use Mp3Error;
+
+impl FrameHeader {
```

Comment:
Why are we doing `impl FrameHeader` in `frame.rs` and not in `header.rs`? This should definitely be moved to `header` IMO.